Packs/CommonTypes/IndicatorFields/indicatorfield-signed.json

```diff
@@ -19,10 +19,7 @@
     "editForm": true,
     "description": "",
     "associatedToAll": false,
-    "associatedTypes": [
-        "File",
-        "URL"
-    ],
+    "associatedTypes": [],
```

Comment:
Don't you think we should keep the signatures data? maybe with a grid or different format but remove it from the file completely?

Reply:
We can, I truly have no opinion in the matter

Reply:
lets add it :)

Reply:
Do you have examples for that data?